CONTRIBUTING.md

```diff
@@ -68,7 +68,7 @@ If you decide to make a contribution, please do use the following workflow:
 To be able to contribute you need at least the following:
 
 - _Git_;
-- _NodeJS_ v14.16 or higher and _NPM_ v7 or higher;
+- _NodeJS_ v14.16 or higher and _NPM_ v8.1.2 or higher;
```

Comment:
The styling of "NodeJS" and "NPM" should be updated here in line with the official stylized version of the names, based on the (current) `<title>`s of the websites.

```suggestion
- _Node.js_ v18 or higher and _npm_ v8.1.2 or higher;
```